.ci/azure-pipelines/matrix.yml

```
@@ -0,0 +1,88 @@
+parameters:
+  templates:
+    Linux: unix.yml
+    macOS: unix.yml
+    Windows: windows.yml
+  
+  Linux:
+    CC: gcc-8
+    FC: gfortran-8
+  macOS:
+    CC: gcc-8
+    FC: gfortran-8
+  Windows:
+    CC: gcc
+    FC: gfortran
+  
+  common:
+    WATS_REPO: WRF-CMake/wats
+    WATS_BRANCH: letmaik/plots
```

Needs a PR for [letmaik/plots](https://github.com/WRF-CMake/wats/tree/letmaik/plots) first.

```suggestion
    WATS_BRANCH: master
```

---

PR at https://github.com/WRF-CMake/wats/pull/3.